kayenta-web/kayenta-web.gradle

```diff
@@ -30,6 +31,7 @@ dependencies {
   api project(':kayenta-objectstore-configbin')
   api project(':kayenta-objectstore-memory')
   api project(':kayenta-orca')
+  api project(':kayenta-oss')
```

Comment:
Spinnaker and Kayenta are Open Source Software (OSS), so when you use this name in your code it becomes overloaded for me.

I think it would be better if `oss` was referred to as `object-storage-service` throughout the code base.

Reply:
I'd include `aliyun` in the name as well, as even something like `object-storage-service` might be confusing.

Reply:
Can i use alicloud-oss ？